.github/workflows/build_and_test.yaml

```diff
@@ -30,14 +29,18 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "macos-latest", "windows-latest"]
-        browser: ["Firefox", "Chrome"]
+        browser: ["Chrome", "Firefox", "Edge", "Safari"]
+        exclude:
+          - os: ubuntu-latest
+            browser: Edge
```

Comment:
I think that this combination is available also

Reply:
Not according to GitHub docs: https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md#browsers-and-drivers